rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
```

Comment:
LGTM. CC @razvan because you know the templating stuff pretty good. Maybe we want to template this here.

Reply:
This is already in operator-templating.

Reply:
Yeah, but looking at https://github.com/stackabletech/operator-templating/blob/main/repositories.yaml only for the product operator, is that correct? We are in operator-rs here.
And: If the answer is "Not worth the effort" that's totally fine by me.

Reply:
Okay how should we proceed then? I think manually updating this should be easy, but it should probably be documented (or be part of the post/pre-release tasks).